automation/test.sh

```diff
@@ -61,7 +61,12 @@ if [ ! -d "cluster-up/cluster/$KUBEVIRT_PROVIDER" ]; then
   exit 1
 fi
 
-export KUBEVIRT_NUM_NODES=2
+if [[ $TARGET =~ sriov.* ]]; then
+  export KUBEVIRT_NUM_NODES=3
```

Comment:
we still have 3 nodes so it will fail 
as we see here
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/5486/pull-kubevirt-e2e-kind-1.17-sriov/1384775716037464064

Reply:
What are you talking about? It needs 3 nodes for the migration to be possible.
What are you nacking here exactly and why?

Reply:
I added this commit (increasing  KUBEVIRT_NUM_NODES) in order to enable testing sriov migration on CI

This PR passed CI few times but did not merged do to how tide works
SRIOV migration test passes on a local setup consistently, its also stable on CI, I think its the first time it failed.
I wouldn't consider it a flake.
I am investigating it on https://github.com/kubevirt/kubevirt/pull/5238

Reply:
This PR adds support for migration
We see migration test is flaky
When i first saw it i thought this PR still want to have 2 nodes, so migration won't run because there were the bugs Or found.
But then understood (correct me if i am wrong), that its intentional that there will be migration which should work and be stable.
So I still left the request changes because, and edited the message, because the test is flaky, 
and its due to this PR which cause regression on the master status (adds a flake)